liveblog.php

```diff
@@ -451,14 +461,14 @@ public static function is_liveblog_post( $post_id = null ) {
 	 * @return bool
 	 */
 	public static function is_viewing_liveblog_post() {
-		return (bool) ( is_single() && self::is_liveblog_post() );
+		return (bool) ( is_singular() && self::is_liveblog_post() );
```

Comment:
This will return true for all post types, including unsupported post types. This needs more precision. Also, possibly a filter - not sure.

Reply:
is_singular will return true for all default single post types. 
Correct me if I am wrong, but for it to return true on a custom post type, the name of the custom post type needs to be sent as a parameter. 

Also, how do you think this conditional should be handled?

Reply:
https://codex.wordpress.org/Function_Reference/is_singular#Custom_Post_Types

Reply:
`is_singular` by default will check if any post type is singular, not just post or pages. So you have to check the `is_signular()` against all of the post types that support LIveblog.